hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/node/DatanodeAdminMonitorImpl.java

```diff
@@ -363,6 +427,14 @@ private boolean checkContainersReplicatedOnNode(DatanodeDetails dn)
     LOG.info("{} has {} sufficientlyReplicated, {} underReplicated and {} " +
         "unhealthy containers",
         dn, sufficientlyReplicated, underReplicated, unhealthy);
+    containerStateByHost.computeIfAbsent(dn.getHostName(),
+            hostID -> new ContainerStateInWorkflow(hostID,
```

Comment:
The indent here again seems too long (8 spaces instead of 4?) and all the `0L` could be pulled onto the same line I think.

Reply:
Fixed.